packages/components/src/Switch/Switch.css

```diff
@@ -1,5 +1,5 @@
 :root {
-  --switch--pipSize: 30px;
+  --switch--pipSize: calc(var(--space-base) * 1.875); /* equivalent to 30px */
```

Comment:
This comment isn't necessary I don't think. 

The danger with a comment like this is if `--space-base` were to change, this would be incorrect.